fabio.properties

```diff
@@ -512,16 +512,18 @@
 #
 # The default is
 #
-# metrics.circonus.apiapp = "fabio"
+# metrics.circonus.apiapp =
```

Comment:
Hmm, if you set a default in `config/default.go` then this needs to be reflected here. Why did you remove it?

should be `metrics.circonus.apiapp = fabio`